src/gui/shellwidget/shellwidget.cpp

```diff
@@ -1,12 +1,13 @@
 #include "shellwidget.h"
 
+#include "helpers.h"
```

Comment:
`clang-format` is new, and I don't like what it is doing to the headers here.

I'm going to change it. Feel free to ignore the header-related format failures.